test/legacy_test/test_activation_op.py

```diff
@@ -4716,9 +4752,9 @@ def test_check_grad(self):
 create_test_act_fp16_class(
     TestTanh, check_prim=True, check_prim_pir=True, enable_cinn=True
 )
-create_test_act_fp16_class(TestTanhshrink)
-create_test_act_fp16_class(TestHardShrink)
-create_test_act_fp16_class(TestSoftshrink)
+create_test_act_fp16_class(TestTanhshrink, check_pir=True)
```

Comment:
貌似本 pr 未做 tanhshrink 的迁移？所以这个 check_pir 应该关闭
```suggestion
create_test_act_fp16_class(TestTanhshrink)
```

Reply:
done